ivy.el

```diff
@@ -3536,6 +3536,25 @@ Don't finish completion."
       (insert (substring (ivy-state-current ivy-last) 0 -1))
     (insert (ivy-state-current ivy-last))))
 
+(defcustom ivy--preferred-re-builders
```

Comment:
I find it a bit contradictory for a user option to have a name with double hyphens.